### ISSUE TITLE
drinking fountains in mapnik rendering

### ISSUE THREAD

**Issue by quasart** [der-stefan/OpenTopoMap#227]
Hello,
At first, thank you for these topographic map !

I'm using them and noticed OSM water points are not displayed, I think this information is pretty important when hiking/riding.

I looked quickly at the style, this feature looks easy to add in `symbols-2.xml ` :

```
    <Rule>
          &maxscale_zoom12;
          &minscale_zoom15;
          <Filter>( ([amenity] = 'drinking_water' and [drinkable] != 'no') or ( [drinkable] = 'yes' ) )</Filter>
          <PointSymbolizer file="symbols-otm/drinking_water.png"/>
      </Rule>
      <Rule>
          &maxscale_zoom16;
          &minscale_zoom17;
          <Filter>( ([amenity] = 'drinking_water' and [drinkable] != 'no') or ( [drinkable] = 'yes' ) )</Filter>
          <PointSymbolizer file="symbols-otm/drinking_water_z15.png"/>
          <TextSymbolizer fontset-name="sans-oblique" size="10" fill="#2020ff" margin="1" halo-radius="1" halo-fill="rgba(255,255,255,0.8)" 
              placement="point" dy="8" wrap-width="100" wrap-before="true">[name]</TextSymbolizer>
      </Rule>
```

I cannot test it, but I can send you a pull request with some png files if you're interested.

I adapted the drinking_water filter from OTM garmin directory.

Thank you !

**Comment by kroko83** [comment]
Hello,

I have the same request, OpenTopoMap is my favorite OSM map when hiking or riding on MTB but I feel it lacks this vital information (water points) when I go out for a long ride in hot weather.
So I also use OpenCycleMap because it displays drinking_water points.

For information, I found that several other tags are closely related and can prove useful when someone is out in an unknown area:
- amenity=drinking_water
- amenity=fountain
- amenity=toilets
- amenity=water_point
- amenity=watering_place (already displayed on OpenTopoMap at zoom 14 and above)
- man_made=water_well
- man_made=water_tap
- natural=spring (already displayed on OpenTopoMap at zoom 14 and above)
- waterway=water_point

**Comment by der-stefan** [comment]
Good point. We will check how those information can be included in the new vector tiles.
